Test/Integration/Model/Transaction/RefundTest.php

```diff
@@ -1,183 +1,66 @@
+use Taxjar\SalesTax\Model\Transaction\Order as TaxjarOrder;
+use Taxjar\SalesTax\Model\Transaction\Refund as TaxjarRefund;
+use Taxjar\SalesTax\Test\Integration\IntegrationTestCase;
+use Taxjar\SalesTax\Util\Fixtures\Sales\CreditmemoBuilder;
+use Taxjar\SalesTax\Util\Fixtures\Sales\OrderBuilder;
 
 /**
  * @SuppressWarnings(PHPMD.CouplingBetweenObjects)
- * @see https://app.hiptest.com/projects/69435/test-plan/folders/419534/scenarios/2587535
  */
-class RefundTest extends \PHPUnit\Framework\TestCase
+class RefundTest extends IntegrationTestCase
 {
-    /**
-     * @var CleanupReservationsInterface
-     */
-    protected $cleanupReservations;
-
-    /**
-     * @var Order
-     */
-    protected $order;
-
     /**
      * @var \Taxjar\SalesTax\Model\Transaction\Order
      */
-    protected $transactionOrder;
+    protected $taxjarOrder;
 
     /**
      * @var \Taxjar\SalesTax\Model\Transaction\Refund
      */
-    protected $transactionRefund;
+    protected $taxjarRefund;
 
     protected function setUp(): void
     {
-        $this->cleanupReservations = Bootstrap::getObjectManager()->get(CleanupReservationsInterface::class);
-        $this->order = Bootstrap::getObjectManager()->get(Order::class);
-        $this->transactionOrder = Bootstrap::getObjectManager()->get('Taxjar\SalesTax\Model\Transaction\Order');
-        $this->transactionRefund = Bootstrap::getObjectManager()->get('Taxjar\SalesTax\Model\Transaction\Refund');
-    }
-
-    /**
-     * @magentoDataFixture ../../../../app/code/Taxjar/SalesTax/Test/Integration/_files/transaction/refund_bundle_partial.php
-     */
-    public function testDefaultRefund()
-    {
-        $order = $this->order->loadByIncrementId('100000001');
-        $creditmemo = $order->getCreditmemosCollection()->getFirstItem();
-        $result = $this->transactionRefund->build($order, $creditmemo);
-
-        if (isset($result['line_items'])) {
-            $lineItems = $result['line_items'];
-
-            $this->assertEquals(2, count($lineItems), 'Number of line items is incorrect');
-            $this->assertEquals(1, $lineItems[0]['quantity'], 'Invalid quantity');
-            $this->assertEquals('24-WG082-blue', $lineItems[0]['product_identifier'], 'Invalid sku.');
-            $this->assertEquals(1, $lineItems[1]['quantity'], 'Invalid quantity');
-            $this->assertEquals('24-WG084', $lineItems[1]['product_identifier'], 'Invalid sku.');
-            $this->assertArrayNotHasKey(2, $lineItems);
-            $this->assertArrayNotHasKey(3, $lineItems);
-        }
+        $this->taxjarOrder = $this->objectManager->get(TaxjarOrder::class);
+        $this->taxjarRefund = $this->objectManager->get(TaxjarRefund::class);
     }
 
     /**
-     * @magentoDataFixture ../../../../app/code/Taxjar/SalesTax/Test/Integration/_files/transaction/refund_simple.php
+     * @magentoDbIsolation enabled
+     * @magentoAppIsolation enabled
      */
     public function testFullRefund()
```

Comment:
Same thing as with some of the test names in the OrderTest class. I think the names of the methods could more clearly indicated the behavior that is being tested.